### ISSUE TITLE
PostgreSQL as a Windows Container

### ISSUE THREAD

**Issue by stellirin** [docker-library/postgres#505]
Hi,

This week I wrote a repository to create a PostgreSQL container image based on the Windows platform. Specifically, the image is based on [microsoft/nanoserver](https://hub.docker.com/r/microsoft/nanoserver/), version `1803` (the latest).

The container image uses the [BigSQL](http://www.openscg.com/bigsql/about/) distribution of PostgreSQL, which is one of the two official distributions found on the [PostgeSQL for Windows](https://www.postgresql.org/download/windows/) download page.

I used the official [Dockerfile](https://github.com/docker-library/postgres/blob/master/10/Dockerfile) and [docker-entrypoint.sh](https://github.com/docker-library/postgres/blob/master/10/docker-entrypoint.sh) as inspiration for building functionality, so with this container image you can (probably) do everything that is possible with the Linux based container image.

I have tested the image against a proprietary enterprise Java application, and all appears to be working well.

I have seen some interest by others for a Windows based PostgreSQL container image, so I'm willing to contribute the files in my repository to this project.

My repository is found at [stellirin/docker-postgres-windows](https://github.com/stellirin/docker-postgres-windows)

Please take a look at my project and provide feedback. No doubt there will be several issues to work through before we can merge something into this project, but it is something I have a strong motivation to achieve.

Some general topics I can think of:

1. Do we really want to use the BigSQL distribution or compile it independently?
1. Are there any Windows specific functionalities we can take advantage of?
1. How can we reconcile the differences in the `Dockerfile` and `docker-entrypoint.cmd`? Should we?

Thanks!

**Comment by tianon** [comment]
Duplicate of https://github.com/docker-library/postgres/issues/324. :wink:
